go/stellar/transform.go

```diff
@@ -105,11 +105,22 @@ func TransformRequestDetails(mctx libkb.MetaContext, details stellar1.RequestDet
 
 // transformPaymentStellar converts a stellar1.PaymentSummaryStellar into a stellar1.PaymentLocal.
 func transformPaymentStellar(mctx libkb.MetaContext, acctID stellar1.AccountID, p stellar1.PaymentSummaryStellar, oc OwnAccountLookupCache, exchRate *stellar1.OutsideExchangeRate) (*stellar1.PaymentLocal, error) {
-	loc, err := newPaymentLocal(mctx, p.TxID, p.Ctime, p.Amount, p.Asset, p.From, p.To, acctID, exchRate)
+	loc, err := newPaymentLocal(mctx, p.TxID, p.Ctime, p.Amount, p.Asset, exchRate)
 	if err != nil {
 		return nil, err
 	}
 
+	isSender := p.From.Eq(acctID)
```

Comment:
this stuff seems new...is it supposed to be part of this change?

Reply:
Looking